src/gui/properties/propertieswidget.cpp

```diff
@@ -414,11 +414,11 @@ void PropertiesWidget::loadDynamicData() {
 
         label_seeds_val->setText(tr("%1 (%2 total)", "%1 and %2 are numbers, e.g. 3 (10 total)")
                                 .arg(QString::number(m_torrent->seedsCount()))
-                                .arg(QString::number(m_torrent->totalSeedsCount())));
+                                .arg(QString::number(m_torrent->totalSeedsCount() + std::max(m_torrent->completeCount(), 0))));
```

Comment:
Maybe it will be better if you modify `TorrentHandle::totalSeedsCount()` (and so on) instead of adjustments in client code?